Platform/ApollolakeBoardPkg/AcpiTables/Dsdt/Sc.asl

```diff
@@ -126,9 +135,12 @@ scope (\_SB.PCI0) {
         Return (0x00140001)
       }
     }
-    Name(_PRF, 0x08)
-    Include("ScPcie.asl")
-    Method(_PRW, 0) { Return(GPRW(_PRF, 4)) }  // Leaf Hill CRB: GPE enable bit for RP2 is 8, can wake up from S4 state
+
+    #define PCIE_WAKE_GPE_BIT  RPW2
+    #include "ScPcie.asl"
+    #undef  PCIE_WAKE_GPE_BIT
+
+    Method(_PRW, 0) { Return(GPRW(RPW2, 4)) }  //GPE enable bit for RP2 is 8, can wake up from S4 state
```

Comment:
This comment doesn't have any relevance any more, should be just "can wakeup from S4 state" like the others?

Reply:
Sure,  Fixed it in new commit.